Jint.Tests/Parser/JavascriptParserTests.cs

```diff
@@ -9,21 +9,21 @@ public class JavascriptParserTests
         [Fact]
         public void ShouldParseThis()
         {
-            var program = new JavaScriptParser().ParseScript("this");
+            var program = new Acornima.Parser().ParseScript("this");
             var body = program.Body;
 
             Assert.Single(body);
-            Assert.Equal(Nodes.ThisExpression, body.First().As<ExpressionStatement>().Expression.Type);
```

Comment:
and another useful global using alias would be mapping now already `Nodes` => `NodeType`, never liked the name missing Type suffix actually 😉